tests/unit/inference/test_inference.py

```diff
@@ -97,7 +98,8 @@ def _hf_model_list() -> List[ModelInfo]:
     current_time = time.time()
 
     # Update the cache if it has expired
-    if (model_data["cache_time"] + cache_expiration_seconds) < current_time:
+    if ((model_data["cache_time"] + cache_expiration_seconds) < current_time) or os.getenv("FORCE_UPDATE_HF_CACHE",
```

Comment:
Thanks for this @raza-sikander - not sure if we should document this anywhere though?

Reply:
This is the only place it is used, so this should be fine to leave.  Thanks for this change.